lib/containers/common.pm

```diff
@@ -116,6 +116,10 @@ sub install_buildah_when_needed {
             zypper_call('in buildah', timeout => 300);
         }
     }
+    if ((script_run 'buildah info') != 1) {
```

Comment:
Didn't you thought about `script_output()` validation instead? In my opinion `buildah info` returning `1` can happen for various reasons.

Reply:
*this change is ready for review* http://aquarius.suse.cz/tests/7419

Reply:
Nice! :+1: